src/command_modules/azure-cli-locationbasedservices/azure/cli/command_modules/locationbasedservices/custom.py

```diff
@@ -0,0 +1,134 @@
+    """Get a Location Based Services Account.
+
+    :param resource_group_name: The name of the Azure Resource Group.
+    :type resource_group_name: str
+    :param account_name: The name of the Location Based Services Account.
+    :type account_name: str
+    :return: LocationBasedServicesAccount
+    :rtype:
+     ~azure.mgmt.locationbasedservices.models.LocationBasedServicesAccount
+    :raises:
+     :class:`ErrorException<azure.mgmt.locationbasedservices.models.ErrorException>`
+     or :class:`ErrorException<knack.util.CLIError>`
+    """
+    # Retrieve ClientRawResponse from get call
+    out = client.get(resource_group_name, account_name, raw=True)
+    # Handle 404 error
+    if out.response.status_code == 404:
+        raise CLIError("The resource 'Microsoft.LocationBasedServices/accounts/" + account_name +
+                       "' under resource group '" + resource_group_name + "' was not found.")
+    return out.output
+
+
+def create_account(client, resource_group_name, account_name, sku_name='S0', tags=None, agree=None):
+    """Create a Location Based Services Account. A Location Based
+    Services Account holds the keys which allow access to the Location
+    Based Services REST APIs.
+
+    :param resource_group_name: The name of the Azure Resource Group.
+    :type resource_group_name: str
+    :param account_name: The name of the Location Based Services Account.
+    :type account_name: str
+    :param sku_name: The name of the SKU, in standard format (such as S0).
+    :type sku_name: str
+    :param tags: Gets or sets a list of key value pairs that describe the
+     resource. These tags can be used in viewing and grouping this resource
+     (across resource groups). A maximum of 15 tags can be provided for a
+     resource. Each tag must have a key no greater than 128 characters and
+     value no greater than 256 characters.
+    :type tags: dict[str, str]
+    :param agree: If true, user agrees to the Preview Terms. Ignore prompt
+     for confirmation. False otherwise.
+    :type agree: bool
+    :return: LocationBasedServicesAccount
+    :rtype:
+     ~azure.mgmt.locationbasedservices.models.LocationBasedServicesAccount
+    :raises:
+     :class:`ErrorException<azure.mgmt.locationbasedservices.models.ErrorException>`
+    """
+    # Prompt for the Preview Terms agreement.
+    warning_msg = 'By creating a Location Based Services account, you agree to the Microsoft Azure Preview Terms.' + \
+                  '\nThe Preview Terms can be found at: ' + \
+                  '\nhttps://azure.microsoft.com/en-us/support/legal/preview-supplemental-terms/'
+    logger.warning(warning_msg)
+
+    if not agree:  # ... in order to pass ScenarioTest
+        response = prompt_y_n('I confirm that I have read and agree to the Microsoft Azure Preview Terms.')
+        if not response:
+            raise CLIError('You must agree to the Microsoft Azure Preview Terms to create an account.')
```

Comment:
This is generally not used anywhere else for create commands.  Is there a particular reason LBS would be different?